lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -522,7 +551,7 @@ class PatternStopContents extends Component<Props, State> {
             </FormGroup>
           </Col>
         </Row>
-        {feedSource.flexUIFeaturesEnabled && pickupDropOffRow}
+        {feedSource.flexUIFeaturesEnabled && this._renderPickupDropOffTypes(false)}
         {feedSource.flexUIFeaturesEnabled && (
```

Comment:
I think based on this we would also never show continuous pickup and dropoff fields for a normal stop as well right? This seems like a legacy issue

Reply:
No, that function does not control rendering of continous pickup/dropoff fields.

Reply:
Right, but it is avoiding rendering the pickup and dropoff for a non-flex feed in the flex branch right?

Reply:
See https://github.com/ibi-group/datatools-ui/pull/889#discussion_r1053364100.